### ISSUE TITLE
re-login

### ISSUE THREAD

**Issue by higuchi-toshio-intec** [conwetlab/ckanext-oauth2#60]
Very thanks for your project.
My execution environment is built with these versions.
- AWS Fargate
  - CKAN [2.9.5]
  - ckanext-oauth2 [0.7.0]
- Amazon ELB (for "https" front-end listener)
- Amazon Cognito

I hava questions for re-login.

I operated my browser like these.
1. I browsed my CKAN top page.
1. I clicked "Log in". My browser displayed "sign in dialog".
1. I entered "Username" and "Password" and click "Sign in".
1. I got success logged in.
1. I Clicked "Logout". My browser displayed "Logged Out" message.
1. I clicked "Log in" again.
1. I got success logged in WITHOUT sign in dialog.

Is this behavior normal?

What should I do if I want to log in with a different account?

regards

**Comment by aitormagan** [comment]
Take into account that when logging out, your are just closing the CKAN session, but no the IdP one. Because of this, when you click on log in, you are redirected to the IdP, and there, you are redirected straightaways to your CKAN instance, as the IdP has an active account and there is no need to ask for credentials.

If you want to log in in a new account, you should log out in your IdP before trying to log in again in CKAN.

Br
Aitor Magán García

**Comment by hbadri3** [comment]
Yeah, this the meaning of SSO. You can reproduce this behavior (healthy actually) when you log in to a website through google, you will get the same flow, so you need to logout from google if you like to request authentication in your website.
As a workaround you might implement the revoke token of your Idb in you extension (customized)
